### ISSUE TITLE
Use precompilation?

### ISSUE THREAD

**Issue by gaurav-arya** [BioTurboNick/AbbreviatedStackTraces.jl#55]
There's quite a wait time until the first error is displayed on my machine, when using `AbbreviatedStackTraces`. As a MWE, I'm frozen on the below screen for some seconds
```julia
julia> x = y
ERROR:
```
before getting
```julia
julia> x = y
ERROR: UndefVarError: `y` not defined
```
Is there any way to make this faster using the precompilation tooling?

**Comment by BioTurboNick** [comment]
I unfortunately can't precompile because it's overwriting Base methods. Though I think most of that comes from accommodating those who want to ensure stack traces aren't abbreviated in non-interactive mode, tbh.

My feeling is the right solution is to integrate it into Base. (Or I suppose hooks to manipulate stack traces before display?) I would appreciate a vote there. https://github.com/JuliaLang/julia/pull/40537

But I'm also more than happy to take a PR if you find a way to thread this needle of no warnings and lower TTFX.

**Comment by BioTurboNick** [comment]
Fixed in v0.1.10.
